src/test/extension/nodeConfigurationProvidersTests.ts

```diff
@@ -29,9 +29,9 @@ describe('NodeDebugConfigurationProvider', () => {
     EnvironmentVars.platform = process.platform;
   });
 
-  describe.skip('logging resolution', () => {
+  describe('logging resolution', () => {
```

Comment:
these tests just weren't updated when we moved from `logging` to `trace` as the log option